docs/topics/proc_debug-logging-nodejs.adoc

```diff
@@ -0,0 +1,162 @@
+[[debug-logging]]
+= Runtime Application Debug Logging
+
+Debug logging is a way to add granular logging to an application that is explicitly enabled when performing debugging. This allows you to keep this logging within your application and only enable it when needed.
+
+
+== Add Debug Logging
+There are several options for debug logging, but this example uses the link:https://www.npmjs.com/package/debug[debug package].
+
+[discrete]
+=== Prerequisites 
+* An application you want to debug. For example, you can use xref:mission-http-api-nodejs[a booster].
+
+[discrete]
+=== Procedure
+
+. Add the `debug` logging definition.
++
+[source,js,options="nowrap",subs="attributes+"]
+----
+var debug = require('debug')('mybooster');
+----
+
+. Add debug statements.
++
+[source,js,options="nowrap",subs="attributes+"]
+----
+app.use('/api/greeting', (request, response) => {
+  const name = request.query ? request.query.name : undefined;
+  //log name in debugging
+  debug('name: '+name);
+  response.send({content: `Hello, ${name || 'World'}`});
+});
+----
+
+== Show Debug Logging on Localhost
+
+Use the the `DEBUG` environment variable when starting your application to enable debug logging.
+
+[discrete]
+=== Prerequisites 
+* An application with debug logging.
+
+[discrete]
+=== Procedure
+
+. Start your application with the `DEBUG` environment variable set to enable debug logging.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ DEBUG=mybooster npm start
+----
++
+NOTE: The `DEBUG` environment variable uses link:https://www.npmjs.com/package/debug#wildcards[wildcards] to filter debugging messages.
```

Comment:
Not sure if this needs to be a note. Makes sense either way.

Reply:
updated to make it normal text